extensions/ql-vscode/src/stories/model-alerts/ModelAlerts.stories.tsx

```diff
@@ -3,7 +3,7 @@ import type { Meta, StoryFn } from "@storybook/react";
 import { ModelAlerts as ModelAlertsComponent } from "../../view/model-alerts/ModelAlerts";
 
 export default {
-  title: "CodeQL Model Alerts/CodeQL Model Alerts",
+  title: "Model Alerts/Model Alerts",
```

Comment:
I've dropped the "CodeQL" bit because we've not added it to other stories (e.g. ModelPacks). I see we have it for the model editor stories but I think we should drop it from there too.